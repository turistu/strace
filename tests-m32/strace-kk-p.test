../tests/strace-kk-p.test