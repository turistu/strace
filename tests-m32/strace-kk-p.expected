../tests/strace-kk-p.expected